notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    search("tell me about an important moment or event in your life")
</pre><p> </p><p>Can we add a final Cleanup section?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
There is not much clean up need to be done. Bucket is not created, it's just download the dataset to current dir and run text embeddings on Vertex AI service